### ISSUE TITLE
Enable and disable option for automatic logout

### ISSUE THREAD

**Issue by NavyaLingampalli** [mickesv/LSRE-ReleasePlanningProject#104]
All the users may not want to automatically logout from the system. Hence an option is provided to enable and disable this feature.

**Comment by NavyaLingampalli** [comment]
enhancement of #103

**Comment by sowrabhmummadi** [comment]
what could be the default option ?

**Comment by nitinparasa** [comment]
default option is setting this option to "enabled" state. Just to ensure the security of the user.

**Comment by NavyaLingampalli** [comment]
Yes. It would be better if the default option is set to enabled state.
